README.md

```diff
@@ -14,7 +14,6 @@ Installation
 Install with composer: `composer require ekapusta/doctrine-phinx-bridge-bundle:^1.0`
 
 Add bundle to AppKernel (and DoctrineMigrationsBundle if it wasn't installed before):
-
 ```
             $bundles = [
```

Comment:
Sorry, I meant to remove 4 spaces before $bundles ))

Then code example will look nice.